include/alpaka/mem/fence/MemFenceGenericSycl.hpp

```diff
@@ -54,17 +54,17 @@ namespace alpaka::experimental
         sycl::accessor<int, 1, sycl::access_mode::read_write, sycl::target::global_buffer> m_global_dummy;
         sycl::accessor<int, 1, sycl::access_mode::read_write, sycl::target::local> m_local_dummy;
     };
-} // namespace alpaka::experimental
+} // namespace alpaka
 
 namespace alpaka::trait
 {
     template<typename TMemScope>
-    struct MemFence<experimental::MemFenceGenericSycl, TMemScope>
+    struct MemFence<MemFenceGenericSycl, TMemScope>
     {
-        static auto mem_fence(experimental::MemFenceGenericSycl const& fence, TMemScope const&)
+        static auto mem_fence(MemFenceGenericSycl const& fence, TMemScope const&)
         {
-            static constexpr auto scope = experimental::detail::SyclFenceProps<TMemScope>::scope;
-            static constexpr auto space = experimental::detail::SyclFenceProps<TMemScope>::space;
+            static constexpr auto scope = detail::SyclFenceProps<TMemScope>::scope;
+            static constexpr auto space = detail::SyclFenceProps<TMemScope>::space;
```

Comment:
Since you are touching this region anyway, could you remove the `ext::oneapi` namespace here, too?

Reply:
The last one is an easy fix. However, all the other changes are more involved as SYCL Accessor relies on the three other mentioned files. If I revert changes on those, most of the SYCL accessor will have to go back in the experimental namespace.